src/styles/blocks/card.scss

```diff
@@ -0,0 +1,135 @@
+.card {
+  height: 406px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid;
+  border-color: #f3f3f3;
+  background-color: white;
+  transition: all 0.3s ease;
+  cursor: pointer;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__product {
+    &-name {
+      padding-top: 40px;
+      line-height: 18px;
+      font-weight: 500;
+      font-size: 12px;
+      color: #060b35;
+      padding: 40px 16px 0;
+    }
+
+    &-code {
+      line-height: 14px;
+      font-size: 10px;
+      font-weight: 400;
+      color: #616070;
+      padding: 4px 16px 0;
+    }
+
+    &-ratings {
+      display: flex;
+      justify-content: space-between;
+      align-items: flex-end;
+      margin: auto;
+      padding-left: 16px;
+      padding-top: 16px;
+      padding-right: 16px;
+      height: 16px;
+      margin-top: 16px;
+    }
+
+    &-price {
+      display: flex;
+      padding-top: 24px;
+      justify-content: space-between;
+      align-items: center;
+      color: #616070;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+    color: #616070;
+
+    &-name {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      padding: 0 16px;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      color: #060b35;
+      line-height: 18px;
+      padding: 0 16px;
+    }
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    right: 16px;
+    font-family: "Roboto", Helvetica, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+
+    &-word {
+      @include center;
+      line-height: 14px;
+      color: #060b35;
+    }
+  }
+
+  &__image {
+    @include center;
+    width: 100%;
+    height: 166px;
+    padding-left: 19px;
+    padding-right: 19px;
+    padding-top: 32px;
+  }
+
+  &__buy {
+    &-button {
+      margin: auto;
+      width: 166px;
+      height: 40px;
+      margin-top: 15px;
+      margin-bottom: 16px;
+      @include center;
+      border-radius: 6px;
+      cursor: pointer;
+      border: 1px solid transparent;
+      text-transform: uppercase;
+      background-color: #00acdc;
+      margin-top: 16px;
+      transition: transform 0.3s ease, border 0.3s ease, background-color 0.3s ease, color 0.3s ease;
```

Comment:
Create a variable for transition-duration.
You can also set them as follow:
`transition-property: ...`
`transition-duration: 0.3s`
https://developer.mozilla.org/ru/docs/Web/CSS/transition